README.md

```diff
@@ -151,6 +152,14 @@ development/application/app_api_pass
 development/application/test
 ```
 
+### Delete secret
+Delete the secret from treasury
+
+```
+> treasury delete development/application/secret_key
+Key development/application/secret_key has been successfully deleted
+```
+
```

Comment:
below in Readme you have examples of policies for S3 and SSM. You need to also add all needed there for handling `delete`

Reply:
sure thing added